apps/website/docs/pipelines/typescript-code-review.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 1
+sidebar_label: TypeScript Code Review
+---
+
+# TypeScript Code Review
+
+The
+[TypeScript Code Review workflow](https://github.com/pagopa/dx/blob/main/.github/workflows/js_code_review.yaml)
+is used to ensure code quality for TypeScript projects by running type checks,
+linting, formatting checks, and test coverage reporting. It leverages `turbo`
+for efficient task orchestration across workspaces.
+
+## How It Works
+
+The workflow performs the following steps:
+
+1. **Checks out the code**: Retrieves the repository's code.
+2. **Sets up Node.js and Turbo**: Configures the Node.js environment and
+   prepares `turbo` for task orchestration using the DX-provided
+   [node-setup](https://github.com/pagopa/dx/tree/main/.github/actions/node-setup)
+   action.
+3. **Installs dependencies**: Ensures all required packages are installed using
+   `yarn` with the `--immutable` flag to guarantee a consistent dependency tree.
+4. **Runs the `code-review` script**: Executes the `code-review` script defined
+   in the root `package.json`, which orchestrates tasks like type checking,
+   linting, formatting checks, and optionally test coverage using `turbo`.
+5. **Uploads coverage reports**: If the `CODECOV_TOKEN` secret is set, uploads
+   test coverage reports to Codecov.
```

Comment:
I think that the workflow file is self-explanatory, so it's not needed to describe here each step. I would briefly explain what the workflow do (run `code-review` script and optionally upload the `coverage` result to codecov.io)